docs/source/quickstart.rst

```diff
@@ -68,6 +94,10 @@ Run it!
 The submit command can take several minutes as it brings up your cluster and downloads your BLAST database.
 **NOTE**: currently you can only have **one** ElasticBLAST search running at a time.
 
+You can also add --sync to the above command-line, in which case elastic-blast will automatically shut 
+down your cluster when it's done.  In this case, it's important that your computer stays powered up and connected 
+to the internet, so that elastic-blast can issue the command to delete the cluster.
```

Comment:
```suggestion
to the internet, so that elastic-blast can issue the command to delete the cluster.


If you are running `elastic-blast --sync` in remote/shared linux server, please consider using `nohup` or a terminal multiplexer (e.g.: `screen` or `tmux`) to keep the process alive in the event of network disconnection or log out.
```

Reply:
Added this.